ssl/ssl_local.h

```diff
@@ -2115,6 +2115,8 @@ typedef enum downgrade_en {
 #define TLSEXT_SIGALG_rsa_pss_rsae_sha384                       0x0805
 #define TLSEXT_SIGALG_rsa_pss_rsae_sha512                       0x0806
 #define TLSEXT_SIGALG_rsa_pss_pss_sha256                        0x0809
+#define TLSEXT_SIGALG_gostr34102012_256_intrinsic               0x0840
```

Comment:
This is really a nit however why are you adding the gostr sigalgs in between rsa_pss ones?

Reply:
sorted by value

Reply:
ohhh...
this list isn't sorted, i miss